spec/unit/hanami/action/rules_spec.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+require "rack"
+
+RSpec.describe Hanami::Action do
+  # Deprecated behavior
```

Comment:
I'm not sure why this is "deprecated behavior". This still works, right? Not every action needs rules. Or am I missing something here?

Reply:
Correct!  This comment was from a WIP implementation of `contract do` :)